web/book/src/reference/syntax/date-time/index.md

```diff
@@ -0,0 +1,46 @@
+# Date and time
+
+You already know that PRQL uses the `@2020-01-01` syntax to declare dates. To
```

Comment:
```suggestion
We already know that PRQL uses the `@2020-01-01` syntax to declare dates. To
```

Tiny point — I've generally tried to avoid the second person, so we can try and get a consistent voice through the docs. (Though have not always been successful!)